state/upgrades.go

```diff
@@ -2246,3 +2246,72 @@ func AddControllerNodeDocs(pool *StatePool) error {
 	}
 	return nil
 }
+
+// AddSpaceIdToSpaceDocs ensures that every space document includes a
+// a sequentially generated ID.
+// It also adds a doc for the default space (ID=0).
+func AddSpaceIdToSpaceDocs(pool *StatePool) (err error) {
+	return errors.Trace(runForAllModelStates(pool, func(st *State) error {
+		// Make sure we do not add a default space to the controller model.
```

Comment:
because?

Reply:
> You're missing a step in `upgrades/operations.go`, where you do it for a version?
The 2.7.0 step (present) invokes this.

Reply:
I have to look through the logic, but we probably actually need this concept in controllers for the HA and/or management spaces.

I'll change it.

Reply:
Done.